conda_smithy/cirun_utils.py

```diff
@@ -7,11 +7,25 @@
 
 from cirun import Cirun
 
-# To get this id, got to https://github.com/organizations/<ORG-NAME>/settings/installations
-# and then click on the Configure button next to Cirun Application, then copy
-# the installation id from the URL, it would look something like:
-# https://github.com/organizations/conda-forge/settings/installations/18453316
-CIRUN_INSTALLATION_ID = os.environ.get("CIRUN_INSTALLATION_ID", 18453316)
+
+@lru_cache
+def get_cirun_installation_id(owner: str) -> int:
+    # This ID needs a token with admin: org privileges.
+    # Hard-code instead for easier use
+    if owner == "conda-forge":
+        return 18453316
+    else:
+        from .github import gh_token, Github
```

Comment:
Any reason to have this import here instead of top-level?

Reply:
Moved to top level